documentation/modules/proc-installing-product.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-install-product-{context}']
+= Installing {ProductName}
+
+Using the xref:proc-product-downloads-{context}[download files], install {ProductName} with the Custom Resource Definitions (CRDs) and RBAC configuration required for deployment.
+
+.Prerequisites
+
+* Installation requires a Kubernetes account with cluster admin credentials.
+
+.Procedure
+
+. Log in to the Kubernetes cluster with cluster admin privileges.
+
+. Modify the installation files to reference a new `kafka` namespace where you will install the {ProductName} Kafka Cluster Operator.
++
+NOTE: By default, the files work in the `myproject` namespace.
++
+* On Linux, use:
++
+
+[source, shell, subs=+quotes]
+----
+sed -i 's/namespace: .\*/namespace: kafka/' install/cluster-operator/*RoleBinding*.yaml
+----
++
+* On Mac, use:
++
+[source, shell, subs=+quotes]
+----
+sed -i '' 's/namespace: .\*/namespace: kafka/' install/cluster-operator/*RoleBinding*.yaml
+----
+
+. Create the new `kafka` namespace for the {ProductName} Kafka Cluster Operator.
++
+[source, shell, subs=+quotes ]
+----
+kubectl create ns kafka
+----
+
+. Create the `my-kafka-project` namespace where you will deploy your Kafka cluster.
++
+[source, shell, subs=+quotes ]
+----
+kubectl create ns my-kafka-project
+----
++
+It is best practice to use namespaces to separate functions.
```

Comment:
```suggestion
It is good practice to use namespaces to separate functions.
```

Also, why mention this _after_ all the faffing on with namespaces? Motivate the need for separate namespaces first by saying it's good practice, then create the namespaces then modify the resources seems to me to be the most logical way to do this.

Reply:
Problem solved. I've moved the note out of the steps to the intro of the procedure: 
In this task you create namespaces in the cluster for your deployment.
It is good practice to use namespaces to separate functions.